pypesto/optimize/optimizer.py

```diff
@@ -584,12 +587,13 @@ def __init__(self, par_sigma0: float = 0.25, options: Dict = None):
             (where the optimum is to be expected)
         options:
             Optimizer options that are directly passed on to cma.
+            maxfevals: maximual number of function evaluations
         """
 
         super().__init__()
 
         if options is None:
-            options = {'maxiter': 10000}
+            options = {'maxfevals': 10000}
```

Comment:
hm, in principle good, however there were some issues with this scipy-internally with the various optimizers, therefore please leave the default unchanged for the moment.

Reply:
Done